src/view/pages/CreateOrder/index.tsx

```
@@ -1,18 +1,18 @@
 import { Tooltip, TooltipContent, TooltipTrigger } from '@/view/components/ui/tooltip'
-import { Table, TableBody, TableCell, TableHead, TableHeader, TableRow, TableCaption } from '@/view/components/ui/table'
+import { Table, TableBody, TableHead, TableHeader, TableRow, TableCaption } from '@/view/components/ui/table'
 import { Button } from '@/view/components/ui/button'
-import { Input } from '@/view/components/ui/input'
 import { Textarea } from '@/view/components/ui/textarea'
 import { Alert, AlertDescription, AlertTitle } from '@/view/components/ui/alert'
+import { Label } from '@/view/components/ui/label'
 import { AddOrderProductForm } from './components/AddOrderProductForm'
 import { Combobox } from '@/view/components/Combobox'
-import { Label } from '@/view/components/ui/label'
+import { Kbd } from '@/view/components/Kbd'
```

ESLint

[error] 'Kbd' is defined but never used.

(@typescript-eslint/no-unused-vars)